sources/engine/Xenko.Graphics/Vulkan/CommandList.Vulkan.cs

```diff
@@ -24,6 +24,7 @@ public partial class CommandList
         private int framebufferAttachmentCount;
         private bool framebufferDirty = true;
         private Framebuffer activeFramebuffer;
+        private Texture dummyTexture;
```

Comment:
Shouldn't this be part of `GraphicsDevice.Vulkan.cs` instead?

Reply:
Yes, it should. Working on another issue with Vulkan (https://github.com/phr00t/xenko/issues/49), so I'm not able to get to this right at the moment.

Reply:
Did it when I was trying to fix that other issue. Help there would be greatly appreciated.